synapse/config/server.py

```diff
@@ -124,6 +126,21 @@ def read_config(self, config):
             for domain in federation_domain_whitelist:
                 self.federation_domain_whitelist[domain] = True
 
+        self.federation_ip_range_blacklist = config.get(
```

Comment:
we ought to follow the example of #5134 and include 0.0.0.0 and `::`, whether they were explicitly listed or not.